clang/include/clang/Frontend/CompilerInvocation.h

```diff
@@ -11,6 +11,7 @@
 
 #include "clang/APINotes/APINotesOptions.h"
 #include "clang/Basic/CodeGenOptions.h"
+#include "clang/Basic/DebugOptions.h"
```

Comment:
Wouldn't a forward declaration be enough in this case?

Reply:
done